### PR TITLE
Comma in tags

### PR DESCRIPTION
**Pull request by EdgarsSalmins**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ The following settings can currently be customized
 * allowDuplicates - defaults to **false**
 * caseSensitive - defaults to **true**
 * readOnly - defaults to **false**
+* singleFieldDelimiter - defaults to **,**
```
Comment (patkleef) [inline README.md]: Can also add the property in the example below:
`
[CultureSpecific]
[UIHint("Tags")]
[GetaTags(AllowSpaces = true, AllowDuplicates = true, CaseSensitive = false, ReadOnly = true)]
public virtual string Tags { get; set; }`

File: src/ClientResources/TagsSelection.js [hunk 2]
```diff
@@ -29,7 +29,9 @@ function (
                 allowDuplicates: this.allowDuplicates,
                 caseSensitive: this.caseSensitive,
                 readOnly: this.readOnly,
+                singleFieldDelimiter : this.singleFieldDelimiter,
                 tagLimit: this.tagLimit !== -1 ? this.tagLimit : null,
+                convertFromOldDelimeter: this.convertFromOldDelimeter,
```
Comment (patkleef) [inline src/ClientResources/TagsSelection.js]: Do you use this property somewhere? (convertFromOldDelimeter)

Comment (EdgarsSalmins) [reply src/ClientResources/TagsSelection.js]: no.. i was... but I removed it because of the logic.. thanks for pointing this...

**Comment by patkleef** [comment]
Closed due inactivity

**Pull request closed without merging**